captcha-google-v2/recaptcha.go

```diff
@@ -88,7 +88,7 @@ func (c *Captcha) Verify(captcha, userInput string) (pass bool) {
 	}
 	cli := &http.Client{}
 	cli.Timeout = 10 * time.Second
-	resp, err := cli.PostForm("https://www.google.com/recaptcha/api/siteverify", map[string][]string{
+	resp, err := cli.PostForm("https://www.recaptcha.net/recaptcha/api/siteverify", map[string][]string{
```

Comment:
This plug-in needs to load Google resources. Before using it, please make sure that the network can fully load related resources.

Maybe it can be used as an environment variable or configuration.